### ISSUE TITLE
Possibility to change video speed

### ISSUE THREAD

**Issue by eriveraa** [RandomEngy/VidCoder#689]
Hi, VidCoder is really a useful program, but is there any functionality to change the video speed?
I would like to convert my videos to 1.5x speed or 2x speed....is that even possible?
Thanks in advance.

**Comment by RandomEngy** [comment]
No, it's not possible. You could ask the HandBrake team to add that functionality but it might be out of scope since it's mainly a transcoder rather than a video editing tool.

**Comment by eriveraa** [comment]
Ok @RandomEngy, thanks!
